src/python/pants/jvm/resolve/coursier_setup.py

```diff
@@ -146,6 +154,23 @@ class CoursierSubsystem(TemplatedExternalTool):
         ),
     )
 
+    def generate_url(self, plat: Platform) -> str:
+        platform = self.url_platform_mapping.get(plat.value, "")
+
+        # The arm64 artifacts are hosted in a different repository than coursier/coursier.
+        # If the user has not overridden the defaults url, we'll switch between the templates
+        # based on platform. For v2.1.0-M5-18-gfebf9838c and v2.0.16-169-g194ebc55c,
```

Comment:
Are these also available at the new location? Or must we special-case getting them from coursier/coursier?

Reply:
I'm just wondering why we need to special-case `self.version not in ("v2.1.0-M5-18-gfebf9838c", "v2.0.16-169-g194ebc55c")` specifically. What is different about those two?